INSTALL.md

```diff
@@ -23,32 +23,56 @@ To build **pokeruby.gba**:
 	make -j4
 
 
-# Mac
-
-Install [**devkitARM**](http://devkitpro.org/wiki/Getting_Started/devkitARM).
-
-Then in **Terminal**, run the following commands.
-
-	xcode-select --install
-
-	export DEVKITPRO=${HOME}/devkitPro
-	echo "export DEVKITPRO=${DEVKITPRO}" >> ~/.bashrc
-	export DEVKITARM=${DEVKITPRO}/devkitARM
-	echo "export DEVKITARM=${DEVKITARM}" >> ~/.bashrc
-
-	git clone https://github.com/pret/pokeruby
-	git clone https://github.com/pret/agbcc
-
-	cd agbcc
-	./build.sh
-	./install.sh ../pokeruby
-
-	cd ../pokeruby
-
-To build **pokeruby.gba**:
-
-	make -j4
-
+# macOS
+
+Installing pokeruby on a Mac requires macOS >= 10.12 (Sierra or higher).
+
+To get pokeruby up and running, run the following commands in Terminal:
+
+Make sure you have the right compilers.
+```
+xcode-select --install
+```
+   
+Download the [devkitPRO pacman](https://github.com/devkitPro/pacman/releases/tag/v1.0.0) package (`.pkg`) and install using the native Mac Installer.
+This will install the devkitPRO package in `/opt/devkitpro`.
+
+You must install devkitARM using
+```
+sudo dkp-pacman -S devkitARM 
+```
+
+Then set environment variables 
+```
+export DEVKITPRO=/opt/devkitpro
+echo "export DEVKITPRO=$DEVKITPRO" >> ~/.bashrc
+export DEVKITARM=$DEVKITPRO/devkitARM
+echo "export DEVKITARM=$DEVKITARM" >> ~/.bashrc
+
+echo "if [ -f ~/.bashrc ]; then . ~/.bashrc; fi" >> ~/.bash_profile
+```
```

Comment:
This step is unnecessary, as `sudo dkp-pacman -S gba-dev` will install `devkit-env` which defines the variables in `/etc/profile.d/devkit-env.sh` (automatically called on bash login).

Reply:
not sure why this is still showing up I removed it